### DISCUSSION TITLE
Dynamic fill color in Box Plot chart

### DISCUSSION THREAD

**Discussion by congdv** [apexcharts/apexcharts.js#3812] [General]
I'm wondering that can we customize the fill color of box plot chart for each box plot?. Like this:
![image](https://user-images.githubusercontent.com/8192210/236551006-73e5304e-e6e2-4739-8dc6-4020c01e5966.png)

I checked in the document, and it seems the Apexchart only support upper and lower fill color only:
```
plotOptions: {
  boxPlot: {
    colors: {
      upper: '#5C4742',
      lower: '#A5978B'
    }
  }
}
```
Not the dynamic for each boxplot.

**Comment by beldogueto** [comment]
Thanks for posting about it! I've been trying to achieve this effect for weeks now, but I think it's not currently possible. If it was added a functionality where you could set the border color and the upper/lower fill colors individually in an array and then have them repeated when over (just like the colors: [...] with lines chart), it would be amazing.

Something like this:

```
stroke: {
    colors: ['#000000', '#2a9d8f', 'etc', 'etc']
},
plotOptions: {
    boxPlot: {
        colors: {
            upper: ['#000000', '#2a9d8f', 'etc', 'etc'],
            lower: ['#000000', '#2a9d8f', 'etc', 'etc']
        }
    }
}
```
